dlio_benchmark/configs/workload/llama_7b.yaml

```diff
@@ -0,0 +1,46 @@
+# 8 node run with 4 GPUs per node and TPSIZE=4 and PPSIZE=8
+model:
+  name: llama_7b
+  type: transformer
+  model_size: 30102
+  num_layers: 32
+  parallelism: 
+    pipeline: 1
+    tensor: 1
+    zero_stage: 1
+  transformer: 
+    vocab_size: 32000
+    hidden_size: 4096
+    ffn_hidden_size: 11008
+
+framework: pytorch
+
+workflow:
+  generate_data: True
+  train: True
+  checkpoint: True
+
+dataset: 
+  data_folder: data/llama_7b/
+  format: mmap_indexed_binary
+  num_files_train: 1
+  num_samples_per_file: 1048576
+  record_length: 2048
+  
+reader: 
+  data_loader: pytorch
+  batch_size: 16
+  read_threads: 1
+  file_shuffle: seed
+  sample_shuffle: seed
+
+train:
+  epochs: 1
+  computation_time: 5 # This is not actual measurement. Just set an interval so that checkpoint every 5 seconds
+  total_training_steps: 5
+
+checkpoint:
+  checkpoint_folder: checkpoints/llama_7b
+  steps_between_checkpoints: 1
+  model_datatype: fp16
+  optimizer_datatype: fp32
```

Comment:
I feel these should be in model. and you missed adding a new line at end of file here.

Reply:
This was not fixed.